modules/cnf-topology-aware-lifecycle-manager-precache-user-spec-images.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_content-type: CONCEPT
+[id="talm-prechache-user-specified-images-concept_{context}"]
+= Pre-caching user-specified images with {cgu-operator} on {sno} clusters
+
+You can pre-cache application-specific workload images on {sno} clusters before upgrading your applications.
+
+You can specify the configuration options for the pre-caching jobs using the following custom resources (CR):
+
+* `PreCachingConfig` CR
+* `ClusterGroupUpgrade` CR
+
+[NOTE]
+====
+All fields in the `PreCachingConfig` CR are optional.
+====
+
+.Example PreCachingConfig CR
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: PreCachingConfig
+metadata:
+  name: exampleconfig
```

Comment:
I think we are missing the "namespace:" field in the example CR. It should be in the metadata: block.
Based on the example CGU below, it should read:     namespace: exampleconfig-ns